src/main/java/com/exasol/adapter/capabilities/MainCapability.java

```diff
@@ -7,18 +7,17 @@
     /**
      * Support projections, i.e. request only a subset of all columns of the table.
      * <p>
-     * This capability contains a subset of SELECTLIST_PROJECTION. It allows projections, if the select list contains an
-     * expression.
-     * </p>
-     * <p>
      * Example: SELECT a FROM t;
      * </p>
      */
     SELECTLIST_PROJECTION,
 
     /**
      * Support expressions in the select list; Additional capabilities are required depending on the expression.
-     *
+     * <p>
+     * This capability contains a subset of SELECTLIST_PROJECTION. It allows projections, if the select list contains an
+     * expression.
+     * </p>
```

Comment:
For consistency within the file:
```suggestion
     * <p>
     * Attention: This capability contains a subset of SELECTLIST_PROJECTION. It allows projections if the select list
     * contains an expression.
     * </p>
     *
```